install/resources/mybb_theme.xml

```diff
@@ -14359,5 +14356,10 @@ $(function() {
 </tr>]]></template>
 		<template name="task_image" version="1800"><![CDATA[<img src="{$mybb->settings['bburl']}/task.php" width="1" height="1" alt="" />]]></template>
 		<template name="smilie" version="1800"><![CDATA[<img src="{$smilie['image']}" alt="{$smilie['name']}" title="{$smilie['name']}" class="smilie smilie_{$smilie['sid']}{$extra_class}"{$onclick} />]]></template>
+	<template name="delete_attachments_button" version="1800"><![CDATA[		
```

Comment:
Can we change this template version to match the next release? I'm happy to merge once that's done 😄 

```suggestion
	<template name="delete_attachments_button" version="1825"><![CDATA[		
```